vizro-core/src/vizro/tables/_dash_table.py

```diff
@@ -19,6 +19,13 @@ def dash_data_table(data_frame: pd.DataFrame, **kwargs):
             "border_top": "1px solid var(--main-container-bg-color)",
             "height": "32px",
         },
+        "style_data_conditional": [
+            {
+                "if": {"state": "active"},
+                "backgroundColor": "var(--state-overlays-selected)",
+                "border": "1px solid var(--state-overlays-selected)",
+            }
+        ],
```

Comment:
Unfortunately can't be moved to CSS due to the data-table using inline style, so I can't make it more specific via pure css without using the !important tag. So this has to live here